chassis/configuration/configuration/basic/default.environment.properties

```diff
@@ -68,17 +68,17 @@ chassis.storage.hibernate.batch_size=50
 # This is a max count of broken sessions which temporary are stored in a database (KeyValue)
 chassis.storage.temporary.data.session.count=50
 
```

Comment:
why are you changing default DB to mysql in the story dedicated to documentation?

H2 should remain default DB